src/emulator/storage/files.ts

```diff
@@ -266,7 +270,7 @@ export class StorageLayer {
     return this._persistence.deleteAll();
   }
 
-  public finalizeUpload(uploadId: string): FinalizedUpload | undefined {
+  public finishUpload(uploadId: string): FinalizedUpload | undefined {
```

Comment:
nit: I know the rest of the file doesnt do this but we should follow the style guide here and add JSDoc documentation to all top-level exported functions, methods, and classes. see go/tsstyle#document-all-top-level-exports-of-modules

You don't have to modify the other methods in this file but for any new public methods, let's make sure to document them properly.

Reply:
Done.